src/app/dataset-block/metadata-block/components/block-header/block-header.component.ts

```diff
@@ -0,0 +1,50 @@
+import { DataHelpers } from "./../../../../common/data.helpers";
+import { DatasetService } from "./../../../../dataset-view/dataset.service";
+import { BaseComponent } from "src/app/common/base.component";
+import { MetadataBlockFragment } from "./../../../../api/kamu.graphql.interface";
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Input,
+    OnInit,
+} from "@angular/core";
+import { Observable } from "rxjs";
+import { NavigationService } from "src/app/services/navigation.service";
+import { DatasetInfo } from "src/app/interface/navigation.interface";
+
+@Component({
+    selector: "app-block-header",
+    templateUrl: "./block-header.component.html",
+    styleUrls: ["./block-header.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BlockHeaderComponent extends BaseComponent implements OnInit {
+    public block$: Observable<MetadataBlockFragment>;
+    @Input() public blockHash: string;
+    @Input() public datasetInfo: DatasetInfo;
+    constructor(
```

Comment:
Not enough empty lines around constructor

Reply:
Fixed